src/index.html

```diff
@@ -11,7 +11,164 @@
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="formField">
+          How old are You?
+          <input 
+            type="number" 
+            name="age" 
+            min="1" 
+            max="100" 
+            value="12"
+          >
+        </label>
+
+        <label class="formField">
+          Full date of birth:
+          <input type="date" name="dateOfBirth">
+        </label>
+
+        <label class="formField">
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="terms" 
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label class="formField">
+          E-mail:
+          <input 
+            type="email" 
+            name="email" 
+            placeholder="email@example.com" 
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="formField">
+          Password:
+          <input 
+            type="password" 
+            name="password" 
+            minlength="5" 
+            maxlength="20"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <span class="formField">
+          Do you love cats?
+          <input 
+            type="radio" 
+            id="yesCat" 
+            name="doYouLoveCats" 
+            value="yes"
+          >
+          <label for="yesCat">Yes</label>
+
+          <input 
+            type="radio" 
+            id="noCat" 
+            name="doYouLoveCats" 
+            value="no"
+          >
+          <label for="noCat">No</label>
+        </span>
+        
+        <label class="formField">
+          What is your favorite color?
+          <input type="color" name="favoriteColor">
+        </label>
+
+        <label class="formField">
+          What time do you go to bed?
+          <input type="time" name="timeToBed">
+        </label>
+
+        <label class="formField">
+          What are your favorite brand of cars?
+          <select 
+            name="favoriteCars" 
+            size="4" 
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="formField">
+          How do you rate our work?
+          <input 
+            type="range" 
+            name="rateOurWork" 
+            min="0" 
+            max="10" 
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <label class="formField">
+          Comments:
+          <textarea 
+            name="comments" 
+            cols="16" 
+            rows="2"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea 
            name="comments" 
            cols="16" 
            rows="2"
          ></textarea>
```

It should fix spaces in textarea field
![image](https://user-images.githubusercontent.com/80700247/218251477-1cfbd381-bb61-4c3a-b6ad-b159c7dd8535.png)

Reply:
Done